src/main/kotlin/org/opensearch/commons/alerting/model/WorkflowRunContext.kt

```diff
@@ -17,7 +17,7 @@ data class WorkflowRunContext(
     val workflowId: String,
     val workflowMetadataId: String,
     val chainedMonitorId: String?,
-    val matchingDocIdsPerIndex: Map<String, List<String>>,
+    val matchingDocIdsPerIndex: Pair<Map<String, List<String>>, List<String>>,
```

Comment:
is this change backward compatible?

can we revert this?

Reply:
fixed this.